src/api/public/apidocs-new/paths/my_notifications.yaml

```diff
@@ -1,6 +1,6 @@
 get:
   summary: List the notifications of the authenticated user.
-  description:  "**(Beta/Unstable)** List user's notifications."
+  description:  "List user's notifications."
```

Comment:
I would add that this endpoint is available in the Unstable version. It has not yet been introduced in any official OBS release.

Reply:
Ok, I'll replace **(Beta/Unstable)** with **(Unstable)** and will add a note in the description; otherwise, I think it's probably not clear what that means.